Makefile

```diff
@@ -143,7 +150,9 @@ tarballs: $(ALL_PKGS_TARGETS)
 
 
 clean:
+	@if [ -t 0 ]; then read -p 'You are about to delete all untracked files! (use [Ctrl+C] to abort) '; fi
 	git clean -xdf
```

Comment:
Maybe it makes sense to remove this rule altogether? It seems that the user is better off just calling `git clean` manually.

Reply:
agreed.

Reply:
sounds good to me, wasn't sure if it was used anywhere